### PR TITLE
first draft of getting started esnext guide

### PR DESCRIPTION
**Pull request by lancelot316**
I need some feedback on the initial draft, of a getting started guide. I used some of the text from the getting started guide from the current Aurelia getting started guide. 

The goal is to create a guide using the CLI as the starting point and building form there.

I'm not sure if I'm on the right path with it yet, and there is still a lot of work to do to get it to a usable state.

### CODE REVIEW

File: guides/getting-started-esnext.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  Before we start writing code. you will need to get setup with the tools we will be using
+  to create the application. We will be using the aurelia comand line tool to create the project
+  structure. The Aurelia CLI will allow to choose between to popular programming language options: ESNext and typescript. In this tutorial we will be using ESnext as our programming language.  
+  
+## Prerequisites
+
+The first thing that you will need to have installed is node.js it can be downloaded 
+from here https://nodejs.org/. Which can be downloaded from the provided link. After following the installation instructions for node. 
+
+The next tool that will need to be installed is yarn, which can
```
Comment (maroy1986) [inline guides/getting-started-esnext.md]: Although Yarn is a pretty good alternative, I think we should still mention NPM here, and its also faster to start if you don't need to install Yarn.

File: guides/getting-started-esnext.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+be downloaded and installed from here https://yarnpkg.com/lang/en/docs/install/
+
+And finally we need to install the Aurelia CLI tool, which can be installed using the 
+yarn tool we installed in the previous step.
```
Comment (maroy1986) [inline guides/getting-started-esnext.md]: Maybe include the actual command to install the CLI so people don't need to search for it outside of this guide.

**Comment by Alexander-Taran** [comment]
Ok.. My previous draft got scrapped..
I come from a position that we can not assume anything about incoming developers
for all we know they know nothing
and every step should be a success worth drinking up
and we don't want to distract them..
Like 1st line of official guide is irrelevant and unsubstantiated..

Aurelia is great..  if you say that - you'll have to explain why.. and compare to other frameworks..
don't go there.. 
let the ppl decide for themselves 

give them something to like
show the product by it's value..

less water.. more 
do 1 - there you have a result
do 2 - there you have another result
maybe with notes for advanced users.. 
want to know how it works - read here (link to docs)

I may be brutal.. but every line.. and every letter should be there for a reason

It is a very hard problem to solve.
And I postponed it w/o notice.. but there is an effort behind the scenes to enable stackblitz for aurelia..
in upcoming weeks.. and the whole strategy of this guide might change..

**Comment by lancelot316** [comment]
I prefer brutal honesty, in a reviewer.  It shortens the feedback cycle, and I am well aware that I don't have as much experience with Aurelia as others in the community. My goal with this guide is to make it easy for newcomers to get started and experiment with the framework, using the tools they will most likely be using on a daily basis, to build things with it.

**Comment by huochunpeng** [comment]
Great starting effect, thank you.

I would say being not very experienced has extra advantage on writing tutorials, as advanced users commonly forgot early struggling in understanding the framework.

I would suggest, not only write a step by step guide, but also explains Aurelia’s design philosophy along the way. That is what people want to know when they try to judge a framework.

For instance:

Why aurelia does not require you to write component by extending some base class, which almost all alternative frameworks do.

How plain JavaScript values (POJO) changes can auto update view. Aurelia make this easy, does not ask you to wrap your plain value in anything, comparing to RxJs and React (setState).

Aurelia’s convention on file names and default view template file name.

Aurelia’s expression support in html template.

All those aurelia’s design decision that attracted us to this framework. They can surely attract more.

**Comment by Alexander-Taran** [comment]
Same comment about being newcomer is a good thing.
Probably everything is fresh..

**Comment by lancelot316** [comment]
@Alexander-Taran just need people to review it, and tell me what if it has errors, and if it hits the target of being an introduction to Aurelia that gets newcomers up and running quickly.

**Comment by Alexander-Taran** [comment]
Good start I think

**Pull request merged**